gapis/api/templates/mutate.go.tmpl

```diff
@@ -27,6 +27,7 @@
     "strings"
 
     "github.com/google/gapid/core/event/task"
+    "github.com/google/gapid/core/log"
```

Comment:
Not sure why the linux build is complaining about this import

Reply:
Done.